openapi-spec/swagger.yaml

```diff
@@ -2737,7 +2737,7 @@ components:
               description: The name of the profile
               minLength: 1
               maxLength: 255
-              pattern: '[A-Za-z0-9_-]'
+              pattern: '^(([a-zA-Z0-9_-]|[a-zA-Z0-9_-][a-zA-Z0-9_-]*[a-zA-Z0-9_-]))*([A-Za-z0-9_-]|[A-Za-z0-9_-][A-Za-z0-9_-]*[A-Za-z0-9_-])$'
```

Comment:
Should we allow any name to start _ or - ? To end with _ or - is also not expected but you can take a call?
https://www.regextester.com/23

Reply:
Thanks, updated to accept [-, _, .] only in between the name